docs/UserGuide.md

```diff
@@ -190,6 +194,7 @@ The table below summarises the add command, format and examples for each contact
 * Salary and commission must be in format ${Number}/hr.<br>
 
 * Price must be in format ${Number}/{quantity}.<br>
+* Employment must be either part-time or full-time.<br>
```

Comment:
good to put "part-time" and "full-time" in quotes to indicate that these are the exact values that `employment` accepts.

Reply:
changed